e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -75,10 +75,7 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
           .should("have.attr", "href")
           .and("eq", standalonePath);
         cy.findByText("Static embedding");
-        cy.findByText(
-          "Embed dashboards, charts, and questions on your app or website with basic filters for insights with limited discovery.",
```

Comment:
I removed some copy assertions, they will be added to the unit/integration tests where it's also going to be faster to get feedback if they fail